### DISCUSSION TITLE
请问IOS Safari浏览器中播放视频无法小窗模式有解决办法吗？

### DISCUSSION THREAD

**Discussion by kksxsyj** [AlistGo/alist#1067] [Q&A] [answered]
![The request is not allowed by the user agent or the](https://user-images.githubusercontent.com/74647878/167392846-bdb9408f-d021-42e6-ae7e-ef17d03afc7c.jpeg)
如图，点小窗会提示这个。

**Comment by xhofe** [comment]
https://alist-doc.nn.ci/docs/setting/frontend#artplayer-whitelist 此项置空 你可以尝试在移动端使用原生播放器而不是artplayer。

**Comment by xhofe** [comment, marked as answer]
https://alist-doc.nn.ci/docs/setting/frontend#artplayer-whitelist 此项置空 你可以尝试在移动端使用原生播放器而不是artplayer。

**Reply by kksxsyj** [reply]
感谢，可以小窗了
